src/python/pants/fs/archive.py

```diff
@@ -116,15 +130,15 @@ def __init__(self, xz_binary_path, xz_library_path):
 
     if not os.path.isdir(xz_library_path):
       raise self.XZArchiverError(
-        "The path {} does not name an existing directory. A directory containing liblzma.so must "
+        "The path {} does not name an existing directory. A directory containing liblzma.{so,dylib} must "
```

Comment:
Let's format in `self. _liblzma_shared_lib_filename` in the error, for clarity (and below)

Reply:
Done!